tox.ini

```diff
@@ -36,75 +30,23 @@ whitelist_externals =
     env
 
 [testenv:server]
-description = Runs all non-Python3-based server unit/functional tests
+basepython = python3.8
```

Comment:
I wonder what this will do on RHEL 8.4, with python 3.6.8?

Reply:
Oh, I thought RHEL 8 had python 3.8 as it's base.  Let's use 3.6 then.